### ISSUE TITLE
fa-pupil or fa-student

### ISSUE THREAD

**Issue by orjanb-ne** [FortAwesome/Font-Awesome#10817]
I woud like an icon for pupils/students. Something like this perhaps?
![pupil-icon-](https://cloud.githubusercontent.com/assets/22000464/24139461/6ed33886-0e1c-11e7-8f49-2254a76456f4.jpg)

**Comment by boylan1340** [comment]
Yes. An icon like this would be great. I work for a school district and  regularly need icons to represent students and student population. I've used the graduation cap and the bell, but could really use something more "pupil" oriented. 

![pupil-desk-white-maroon-circle](https://cloud.githubusercontent.com/assets/28788259/26215088/c8777054-3bb3-11e7-9fdb-0bdaf819c77e.png)

Thanks!

**Comment by tagliala** [comment]
Please note that this example is too complex to fit 16px icon size

**Comment by boylan1340** [comment]
Thanks for the feedback! What do you think of this version?

![pupil-at-desk icon-16x16](https://cloud.githubusercontent.com/assets/28788259/26223692/0d396f10-3bd4-11e7-94b6-6f41532242ab.png)
![pupil-at-desk icon-100x100](https://cloud.githubusercontent.com/assets/28788259/26223691/0d38f210-3bd4-11e7-99d8-d34832d79067.png)

**Comment by boylan1340** [comment]
or maybe without the chair back?
![pupil-at-desk-v2-16x16](https://cloud.githubusercontent.com/assets/28788259/26223825/a9f932fe-3bd4-11e7-9736-87f4000985b3.png)
![pupil-at-desk-v2-100x100](https://cloud.githubusercontent.com/assets/28788259/26223826/a9fc01b4-3bd4-11e7-8358-da2d31773756.png)

**Comment by radubr** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by sensibleworld** [comment]
Hey there!

We've made the decision to move our icon requests flow to **GitHub Discussions**, as opposed to issue tracking. No more +1s or thumbs-ups needed. There's an upvote system built in.

As part of this transition, we are closing existing icon requests with lower comment or reaction counts, but **we are not saying no** to your request. (We know some of these have been open since before I had gray hair. And I've had gray hair for a LONG time.)

**Could you do us a favor? If this icon request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
→ https://github.com/FortAwesome/Font-Awesome/discussions/new?category=icon-requests

And listen. We know this is annoying. We would batch convert them to discussions if we could, but Github's removed that ability earlier this year. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worries at all. I love doing things twice.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. This request already HAS an auto-response saying that you're refining the workflow. What do you mean you're refining your refinement? That's a bunch of bull-phooey.
* No biggie. Thanks for letting us know what's up.
